woocommerce/woocommerce-android
# Analytics Hub - Milestone 1

Summary
==========
Fix issue #7570 by integrating the original Analytics Hub trial code all with the following PR work:

1. [Issue/7572 analytics custom date range](https://github.com/woocommerce/woocommerce-android/pull/7583)(https://github.com/woocommerce/woocommerce-android/pull/7583)
2. [Remove Analytics feature flag. Show only Milestone 1 features in release.](https://github.com/woocommerce/woocommerce-android/pull/7607)
3. [feat: add performance monitoring for analytics hub](https://github.com/woocommerce/woocommerce-android/pull/7637)
4. [Fix percentage labels in Analytics Hub](https://github.com/woocommerce/woocommerce-android/pull/7640)
5. [Add "view analytics" button](https://github.com/woocommerce/woocommerce-android/pull/7548)
6. [feat: remove analytics from bottom bar](https://github.com/woocommerce/woocommerce-android/pull/7549)
7. [fix: unit tests execution](https://github.com/woocommerce/woocommerce-android/pull/7561)
8. [Fixed the Today & Yesterday date ranges](https://github.com/woocommerce/woocommerce-android/pull/7647)
9. [Use sp for the TagView so it handles large fonts](https://github.com/woocommerce/woocommerce-android/pull/7648)
10. [Analytics hub tracks](https://github.com/woocommerce/woocommerce-android/pull/7649)
11. [Top performers with date period](https://github.com/woocommerce/woocommerce-android/pull/7626)
12. [Issue/analytics large font revenue](https://github.com/woocommerce/woocommerce-android/pull/7664)

:warning: All newly introduced code here was reviewed and touched by multiple hands. The main objective of this PR is to review the feature instead of the code solution since this PR is way too big for that.

How to Test
==========
Run a full usage test scenario around the Analytics Hub, paying attention to the following aspects of the feature:
- Verify that the data displayed there reflects what the `wp-admin` Analytics displays
- Verify that only the Report and Orders cards are displayed, showing no Product related stats
- Verify that there's no `See Report` button in any place of the Analytics Hub
- Verify that the Custom date range selector works as expected

Update release notes:
